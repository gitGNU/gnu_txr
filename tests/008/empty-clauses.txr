@(bind a ("a" "b"))
@(output)
@(repeat)
@a
@(first)

@(end)
@(repeat)
@a
@(first)
@(end)
@(repeat)
@a
@(first)
f@a
@(end)
@(rep)@a@(first)@(end)
@(end)
