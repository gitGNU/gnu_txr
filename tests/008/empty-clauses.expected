
b
b
fa
b
b
